kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -640,7 +640,11 @@ public Subscription reactivateSubscription(Subscriber subscriber, Subscription s
         subscription.setStatus(SubscriptionStatus.ACTIVE);
         subscription.setDeactivationReason(null);
         subscriptionService.updateStartDate(subscription, dateTime);
-        subscriptionService.deleteBlockedMsisdn(subscriber.getMother().getId(), null, subscriber.getCallingNumber());
+        if (subscriber.getMother() != null) {
```

Comment:
add nnull check above previous line, since we refer getId from mother in that line too